flexget/plugins/clients/rtorrent.py

```diff
@@ -263,6 +267,11 @@ def load(self, raw_torrent, fields=None, start=False, mkdir=True):
             # TODO: What are the escaping requirements? re.escape works differently on python 3.7+
             params.append(f'd.{key}.set={re.escape(str(val))}')
 
+        # Set custom fields
+        for key, val in custom_fields.items():
+            # Values must be escaped if within params
+            params.append(f'd.custom.set={re.escape(str(key))},{re.escape(str(val))}')
```

Comment:
I'm not sure what escaping rules are needed for the rtorrent api, but I doubt it this is the exact right way. `re.escape` is meant to escape characters that have special meanings in regex. A quick peek at the api guide makes me think maybe just surrounding the key and value with double quotes is enough?
```suggestion
            params.append(f'd.custom.set="{key}","{val}"')
```

Reply:
I've got the `re.escape` used in a couple places. I did the escaping that way because that is what was used already for the fields, and thought I would keep it consistent. However, I will test with just quoting on my parts and see how this goes. If everything works out, then I will commit that.

Reply:
@gazpachoking I have updated all the of the `re.escape` calls used for custom fields to be quoted and to escape double quotes with a `\"` character. I believe this should do the trick. I did leave the re.escape from the fields code that was there before I started my merge since I didn't want to change how things currently work. However, I am happy to make that change if you think it is appropriate.

Thanks for the help!